libenkf/src/enkf_main.c

```diff
@@ -1534,12 +1530,11 @@ void enkf_main_submit_jobs( enkf_main_type * enkf_main ,
 
 
 /**
-  If all simulations have completed successfully the function will
-  return true, otherwise it will return false.
+  The function will return number of non-failing jobs.
```

Comment:
I guess it should be, but the actual thing being counted is "Total - (explicitly failed)", so I called it non-failing.